elements/pl-code/pl-code.py

```diff
@@ -93,22 +110,21 @@ def parse_highlight_lines(highlight_lines: str) -> Optional[list[int]]:
     lines = []
     components = highlight_lines.split(",")
     for component in components:
-        component = component.strip()
+        component = component.replace(" ", "").split("-")
```

Comment:
Rewriting the contents of this function was technically unnecessary, but looking at this again it was confusing enough that I figured it was better just to redo it.

Reply:
Nice! Should we add some tests while we're here?

Reply:
No reason not to! Might have to break this into a couple files but not a huge deal.